common/containers/Tabs/SendTransaction/index.tsx

```diff
@@ -488,21 +489,23 @@ export class SendTransaction extends React.Component<Props, State> {
       return;
     }
 
-    try {
-      const cachedFormattedTx = await this.getFormattedTxFromState();
-      // Grab a reference to state. If it has changed by the time the estimateGas
-      // call comes back, we don't want to replace the gasLimit in state.
-      const state = this.state;
-      gasLimit = await nodeLib.estimateGas(cachedFormattedTx);
-      if (this.state === state) {
-        this.setState({ gasLimit: formatGasLimit(gasLimit, state.unit) });
-      } else {
-        // state has changed, so try again from the start (with the hope that state won't change by the next time)
-        this.estimateGas();
+    if (this.props.wallet) {
+      try {
+        const cachedFormattedTx = await this.getFormattedTxFromState();
```

Comment:
getFormattedTxFromState requires this.props.wallet, otherwise it throws `"Cannot read property 'getAddress' of null"`